examples/falcon_host_details.rs

```diff
@@ -90,7 +90,7 @@ pub struct ApiError(pub String);
 
 impl fmt::Display for ApiError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "Application Errors: {}", self.to_string())
+        write!(f, "Application Errors: {self}")
```

Comment:
I don't think this will work, try:

```suggestion
        write!(f, "Application Errors: {}", self.0)
```

This is due to the Self type not implementing one trait, but the String does.